src/components/Modal/BaseModal.js

```diff
@@ -119,18 +119,15 @@ class BaseModal extends PureComponent {
                 hasBackdrop={this.props.fullscreen}
                 coverScreen={this.props.fullscreen}
                 style={modalStyle}
-
-                // When `statusBarTranslucent` is true on Android, the modal fully covers the status bar.
-                // Since `windowHeight` doesn't include status bar height, it should be added in the `deviceHeight` calculation.
-                deviceHeight={this.props.windowHeight + ((this.props.statusBarTranslucent && StatusBar.currentHeight) || 0)}
+                deviceHeight={this.props.windowHeight}
                 deviceWidth={this.props.windowWidth}
                 animationIn={this.props.animationIn || animationIn}
                 animationOut={this.props.animationOut || animationOut}
                 useNativeDriver={this.props.useNativeDriver}
                 hideModalContentWhileAnimating={this.props.hideModalContentWhileAnimating}
                 animationInTiming={this.props.animationInTiming}
                 animationOutTiming={this.props.animationOutTiming}
-                statusBarTranslucent={this.props.statusBarTranslucent}
+                statusBarTranslucent
```

Comment:
Is this work merged? Currently I don't find any usages of the `statusBarTranslucent`, this is why I removed the prop as I didn't think it would be useful anymore.

Reply:
Sounds good, will add it back for now